### DISCUSSION TITLE
Synchronous Janusgraph cache

### DISCUSSION THREAD

**Discussion by ChathuraRA** [JanusGraph/janusgraph#2971] [Q&A]
In our current usage, we are required to have janusgraph cluster. But it's giving a lack of performance when we disable the cache. In a cluster, we are using Cassandra as backend storage. But when we enabled the cache, we were having a problem with the data consistency. Is there have any way to synchronize cluster cache, or is there have any way to add an external cache option like Redis cluster or something like that.

**Comment by vtslab** [comment]
See the similar discussion at [https://lists.lfaidata.foundation/g/janusgraph-users/topic/janusgraph_database_cache_on/88877425?p=,,,20,0,0,0::recentpostdate/sticky,,,20,2,0,88877425,previd=1644552494121885340,nextid=1641900165208995450&previd=1644552494121885340&nextid=1641900165208995450](https://lists.lfaidata.foundation/g/janusgraph-users/topic/janusgraph_database_cache_on/88877425?p=,,,20,0,0,0::recentpostdate/sticky,,,20,2,0,88877425,previd=1644552494121885340,nextid=1641900165208995450&previd=1644552494121885340&nextid=1641900165208995450)
